Source: Upstream.
Reason: The sip shit broke again.
Upstream: cf. source.

diff -urN pykde4-4.2.85.old/python/pykde4/sip/kdecore/kdatetime.sip pykde4-4.2.85/python/pykde4/sip/kdecore/kdatetime.sip
--- pykde4-4.2.85.old/python/pykde4/sip/kdecore/kdatetime.sip	2009-05-08 12:11:40.514046575 +0400
+++ pykde4-4.2.85/python/pykde4/sip/kdecore/kdatetime.sip	2009-05-08 12:13:25.594046724 +0400
@@ -161,19 +161,3 @@
     static void             setSimulatedSystemTime (const KDateTime& newTime);
     static KDateTime        realCurrentLocalDateTime ();
 };   // KDateTime
-
-
-class KSystemTimeZones_Simulated : KSystemTimeZones /Abstract/
-{
-%TypeHeaderCode
-#include <kdatetime.h>
-%End
-
-public:
-    static KTimeZone        realLocalZone ();
-    static KTimeZone        local ();
-    static void             setLocalZone (const KTimeZone& tz);
-    static bool             isSimulated ();
-};   // KSystemTimeZones_Simulated
-
-
